backend/routes/project.js

```diff
@@ -66,6 +67,11 @@ router.post(
     const projectName = project.projectName;
 
     proposal.signer = res.locals.signer; // signer
+    if (processing.includes(proposal.signer)) {
+      res.status(400).send("Please try again later");
```

Comment:
Looking at the logic... this looks like a backoff so that if the signer can't make more than 2 requests at the same time. 

Unsure if this is a kosher implementation for the problem but this seems like a simple and clean solution. Great!